code/modules/clothing/under/miscellaneous.dm

```diff
@@ -728,3 +728,12 @@
 	icon_state = "hostanclothes"
 	item_state = "hostanclothes"
 	item_color = "hostanclothes"
+
+/obj/item/clothing/under/rank/iaa

```

Comment:
I'll do a discord poll for the two icons if you really want to be annoying about it